test/run-e2e-tests.sh

```diff
@@ -0,0 +1,236 @@
+#!/bin/bash
+###############################################################################
+# Copyright (c) Red Hat, Inc.
```

Comment:
only leave the ocm copy right here.

if it is used in githubaction, we should update the action part. We already start a kind in the action already, can we start to kind using action? and check in the script on whether we need to start kind or not.

Reply:
Fixed https://github.com/open-cluster-management-io/addon-framework/pull/103/commits/c4926eef8416962f3cbc2cd234f273302f445c03